app/components/categories.hbs

```diff
@@ -1,8 +1,8 @@
 <div class="menu-items">
-  {{#each menuItems as |item|}}
-  <LinkTo @route="categories.category" @model={{item.name}}>
-    <div class="{{item.name}} menu-item">{{item.name}}
-    </div>
-  </LinkTo>
+  {{#each @categories as |item|}}
```

Comment:
I can comment on lines with just the keyboard! yay!

I changed menuItems -> @categories because menuItems didn't exist afaict